### PR TITLE
Swaggert docs

### PR DESCRIPTION
**Pull request by boehsermoe**
Just a short implementation of swagger for API definition.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -12,8 +12,40 @@ import { graphqlExpress, graphiqlExpress } from 'apollo-server-express';
 import { makeExecutableSchema } from 'graphql-tools';
 import resolvers from './graphql/resolvers';
 import typeDefs from './graphql/schema';
+import swagger from 'express-swagger-generator';
 
 let app = express();
+
+const expressSwagger = swagger(app);
```
Comment (pkarw) [inline src/index.js]: Maybe this should be optional - enabled by the config based property - not sure if everybody are willing to have swagger on?

**Comment by pkarw** [comment]
That's cool. It would be awesome to extend it by the data format notes as explained in the https://github.com/DivanteLtd/vue-storefront-integration-boilerplate/blob/master/1.%20Expose%20the%20API%20endpoints%20required%20by%20VS/Required%20API%20specification.md

File: src/index.js (hunk 1 above)
Comment (boehsermoe) [reply src/index.js]: Did some refactoring. currently only enable/disable via config, for later we can load the swagger options also from the config.

File: src/api/index.js [hunk 2]
```diff
@@ -61,5 +61,37 @@ export default ({ config, db }) => {
     }
 	}
 
+	if (config.swagger && config.swagger.enabled) {
```
Comment (pkarw) [inline src/api/index.js]: it should be added to `config/default.json` with `swagger.enabled=false` default value. Thanks!

**Comment by pkarw** [comment]
Closing due to long inactivity time

**Pull request closed without merging**